cfgov/unprocessed/js/modules/SkipNav.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Applies a focus state to the main content when a user clicks the skip navigation link
+ * Improves the usability of the skip navigation link across browsers & devices
+ */
+
+function SkipNav() {
+	const skip = document.getElementById( 'skip-nav' );
+	const mainContent = document.getElementById( 'main' );
+
+	skip.addEventListener( 'click', function( e ) {
+    e.preventDefault();
+    mainContent.setAttribute( 'tabindex', '-1' );
```

Comment:
I don't _think_ so, since it is removing `main` from the tab index completely. We could probably set the `tabindex` to `-1` in the markup itself, but this limits it to skip link users.

Reply:
Yeah what would be the potential downside to just setting it in the markup?